src/core/server/deprecations/README.mdx

```diff
@@ -0,0 +1,194 @@
+---
+id: kibUsageCollectionPlugin
+slug: /kibana-dev-docs/services/deprecations-service
+title: Core Deprecations service
+summary: The Deprecations service helps surface deprecated configs and features for plugins to our users
+date: 2021-05-03
+tags: ['kibana','dev', 'contributor', 'api docs']
+---
+
+# Core Deprecations service
+This guide is written in the format of a FAQ to help you get started using the deprecations service.
+For more details on the service contract we've documented all the service apis and properties under the deprecation service docs generated by in core.
+
+## What is the deprecations service?
+The deprecations service provides a way for the Kibana platform to communicate deprecated features and configs with its users. These deprecations are only communicated if the deployment is using these features, allowing for a user tailored experience for upgrading the stack version.
+
+## Where are deprecations displayed to users?
+The Upgrade Assistant (UA) in kibana is consuming the core deprecations service in order to surface plugin deprecations. Each deprecation should have manual steps that we display in the UI to fix the deprecation an API to automatically resolve it.
+
+To check your plugin deprecations go to the UA interface and click on `view deprecations` under the kibana section.
```

Comment:
```suggestion
To check your plugin deprecations, go to the UA interface via **Stack Management > Stack > Upgrade Assistant** and click on `View deprecations` under the Kibana section.
```

Reply:
I think it's also worth mentioning that UA is currently not displaying deprecations by default. A dev will have to change `UA_READONLY_MODE` in [constants.ts](https://github.com/elastic/kibana/blob/master/x-pack/plugins/upgrade_assistant/common/constants.ts#L22) to `false` in order for the deprecations to display.